setup.sh

```diff
@@ -8,19 +8,15 @@ echo 'Installing macOS packages...'
 brew update
 brew install node
 brew install mongodb
-brew install grunt
```

Comment:
For ethan's computer, `yarn dev` failed since grunt wasn't installed. 

Rather than installing grunt-cli, it looks like we should actually [add grunt to our devDependencies](https://gruntjs.com/getting-started#installing-grunt-and-gruntplugins), which will properly install grunt rather than using an outdated [`brew install grunt`](http://brewformulas.org/grunt)